src/content/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import FaFacebookSquare from 'react-icons/lib/fa/twitter-square';
-import FaTwitterSquare from 'react-icons/lib/fa/facebook-square';
+import FaFacebookSquare from 'react-icons/lib/fa/facebook-square';
+import FaTwitterSquare from 'react-icons/lib/fa/twitter-square';
```

Comment:
😂 looks great, thanks!

Reply:
I was a bit confused as to why they were swapped for a bit. :stuck_out_tongue: